.github/workflows/github-release.yml

```diff
@@ -22,81 +38,107 @@ concurrency:
+        with:
+          arch: ${{ matrix.arch }}
+          cpu: ${{ matrix.cpu }}
+          node_tag: ${{ inputs.node_tag }}
+          signer_tag: ${{ inputs.signer_tag }}
+          signer_docker_tag: ${{ inputs.signer_docker_tag }}
+          is_node_release: ${{ inputs.is_node_release }}
 
   ## Runs when the following is true:
-  ##  - tag is provided
-  ##  - workflow is building default branch (master)
+  ##  - either node or signer tag is provided
   create-release:
     if: |
-      inputs.tag != ''
+      inputs.node_tag != '' ||
+      inputs.signer_tag != ''
     name: Create Release
     runs-on: ubuntu-latest
     needs:
       - build-binaries
     steps:
-      ## Downloads the artifacts built in `create-source-binary.yml`
-      - name: Download Artifacts
-        id: download_artifacts
-        uses: actions/download-artifact@65a9edc5881444af0b9093a5e628f2fe47ea3b2e # v4.1.7
-        with:
-          pattern: ${{ inputs.tag }}-binary-build-*
-          path: release
-          merge-multiple: true
-
-      ## Generate a checksums file to be added to the release page
-      - name: Generate Checksums
-        id: generate_checksum
-        uses: stacks-network/actions/generate-checksum@main
+      ## Creates releases
+      - name: Create Release
+        uses: stacks-network/actions/stacks-core/release/create-github-releases@main
         with:
-          artifact_download_pattern: "${{ inputs.tag }}-binary-build-*"
-
-      ## Upload the release archives with the checksums file
-      - name: Upload Release
-        id: upload_release
-        uses: softprops/action-gh-release@69320dbe05506a9a39fc8ae11030b214ec2d1f87 #v2.0.5
-        env:
-          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
-        with:
-          name: Release ${{ inputs.tag || github.ref }}
-          tag_name: ${{ inputs.tag || github.ref }}
-          draft: false
-          prerelease: true
-          fail_on_unmatched_files: true
-          target_commitish: ${{ github.sha }}
-          generate_release_notes: true
-          files: |
-            release/*.zip
-            CHECKSUMS.txt
+          node_tag: ${{ inputs.node_tag }}
+          node_docker_tag: ${{ inputs.node_docker_tag }}
+          signer_tag: ${{ inputs.signer_tag }}
+          signer_docker_tag: ${{ inputs.signer_docker_tag }}
+          is_node_release: ${{ inputs.is_node_release }}
+          is_signer_release: ${{ inputs.is_signer_release }}
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
 
   ## Builds arch dependent Docker images from binaries
   ##
   ## Runs when the following is true:
-  ##  - tag is provided
-  ##  - workflow is building default branch (master)
+  ##  - either node or signer tag is provided
   docker-image:
     if: |
-      inputs.tag != ''
+      inputs.node_tag != '' ||
+      inputs.signer_tag != ''
     name: Docker Image (Binary)
-    uses: ./.github/workflows/image-build-binary.yml
+    runs-on: 'ubuntu-latest'
```

Comment:
any specific reason this is quoted, but not elsewhere?

Reply:
Not any, might have copy-pasted it from somewhere without realizing. Fixed